### PR TITLE
Merge milestone 1 report

### PR DESCRIPTION
**Pull request by dbaslan**
Can we merge to main to get the most up-to-date version of the milestone 1 report? (Report is almost finished, needs a few minor additions)

### CODE REVIEW

**Review by sadikkuzu — CHANGES_REQUESTED** [review]
This branch is out-of-date with the base branch
Merge the latest changes from main into this branch.

image

File: reports/m1_group1.md [hunk 1]
```diff
@@ -104,109 +163,206 @@ Mention any additional task you have performed that is notlisted above.
...
+     * Management-related significant issues:
+     #1, #3, #15, #38, #44, #46, #47, #58, #59, #64, #68, #69, #71, #72, #73, #84, #87, #92
  * Pull requests:
-You have created, merged, and reviewed. Please also briefly summarizewhat the conflict was (if you had any) and how it was resolved, regarding the pullrequests you have reviewed.
+ 1. Reviewed [pull request #93](https://github.com/SWE574-Fall2023-Group1/SWE574-Fall2023-G1/pull/93)
  * Additional information:
-Mention any additional task you have performed that is notlisted above.
-#### Caner
+1. Moderated regular meetings on Discord
+2. Communicated with the customer for feedback on requirements and workflow,
```
Comment (sadikkuzu) [inline reports/m1_group1.md:278]: comma in the end

**Comment by sadikkuzu** [comment]
Btw, my part hasn't finished yet. I'll add more items soon.

**Review by ckertam — CHANGES_REQUESTED** [review]
Let's wait until everyone has done with the report. I will make some changes too. Also, as I know @mert-aydin will make some changes too.

**Comment by sadikkuzu** [comment]
pre-commit.ci autofix

**Comment by sadikkuzu** [comment]
@dbaslan in hackmd, could you please pull from github after pre-commit.ci's auto-fixes?

**Comment by sadikkuzu** [comment]
> This branch is out-of-date with the base branch Merge the latest changes from main into this branch.
> 
> image

@dbaslan if you are available now, we can look at it together in discord.

**Comment by dbaslan** [comment]
Pulled pre-commit's autofixes, merged from main to reports, passed autocheck, awaiting review

**Review by sadikkuzu — COMMENTED** [review]
@dbaslan hi,
I've completed my individual-contributions part.
Could you please push the file from HackMD to GitHub?
Thanks in advance 🙏🏼

**Comment by dbaslan** [comment]
Made final adjustments, awaiting review.

**Review by sadikkuzu — APPROVED** [review]
LGTM 👍🏼

**Review by ckertam — APPROVED** [review]
LGTM! 🥂

**Review by oguzsenna — APPROVED** [review]
Thanks for the effort

**Comment by sadikkuzu** [comment]
Later on, let's add issue number at the end of commit messages.
(Here, for example, `Add final adjustments #64`)
image

**Pull request merged**